R/awsjavasdk-internal.R

```diff
@@ -17,8 +17,9 @@
 .app_dir <- rappdirs::app_dir("awsjavasdk", version = packageVersion("awsjavasdk"))
 
 .onAttach <- function(libname, pkgname) {
+  set_sdk_file_root()
   if (!aws_sdk_present()) {
-    packageStartupMessage("awsjavasdk did not detect its own copy of the AWS SDK; you must execute install_aws_sdk() in order to download the SDK and for the package to work as expected")
+    packageStartupMessage(paste0("awsjavasdk did not detect its own copy of the AWS SDK. You must execute install_aws_sdk() in order to download the SDK and for the package to work as expected. If you've previously installed to a non-standard location, call set_sdk_file_root({{your alt_install root}}, confirm = TRUE) to point this package to that location and then run awsjavasdk::load_sdk()"))
```

Comment:
@gaborcsardi and @jeroenooms do you know any better way of installing a bunch of jar files in an R package? I'm having the same struggle with CRAN policies as Russel here: the AWS jar files are pretty big (~5 - 40Mb depending on how many AWS services you want to include), so cannot put on CRAN.

I was thinking about creating a drat repo on GH, but installation from there also requires extra work from the user at installation time. I'm thinking about a way to automatically download the libs when `R CMD build` runs instead of asking the user to download/install the jars or to provide a helper function so that they have to only confirm that they are OK with this.

In short, any nicer (from the user's perspective) way to handle this? Eg is it possible/legal to use a Makefile to download the jars and add to the package? Or should I just get along with this approach so that users/scripts have to do others things as well besides `install.packages` eg to install my KMS and Kinesis R packages?